Library/Formula/lldpd.rb

```diff
@@ -20,6 +20,8 @@ class Lldpd < Formula
   depends_on "jansson"  if build.with? "json"
 
   def install
+    (var/"run").mkpath
```

Comment:
Maybe we should add an audit check to unify this. We've mixed and matched this between postinstall and install all over the place.